Modules/hpe3par_snapshot.py

```diff
@@ -194,6 +207,26 @@
         storage_system_password="{{ storage_system_password }}"
         state=absent
         snapshot_name="{{ snapshot_name }}"
+
+    - name: Create schedule my_ansible_sc
+      hpe3par_snapshot:
+        storage_system_ip="{{ storage_system_ip }}"
```

Comment:
Please replace the values here with real world values

Eg,
snapshot_name: volume_snapshot,
...